### PR TITLE
Replace jQuery UI Sortable with SortableJS

### PR DESCRIPTION
**Pull request by gagnieray**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -95,6 +95,12 @@ function scripts() {
     .pipe(concat('galette-chartjs.bundle.min.js'))
     .pipe(gulp.dest(_dir));
 
+  sortablejs = gulp.src([
+        './node_modules/sortablejs/Sortable.min.js',
+  ])
+    .pipe(concat('galette-sortablejs.bundle.min.js'))
+    .pipe(gulp.dest(_dir));
+
   return merge(main, chartjs);
```
Comment (trasher) [inline gulpfile.js:104]: Should probably be changed to:
```suggestion
  return merge(main, chartjs, sortablejs);
```

**Comment by trasher** [comment]
Anyways, I'll merge this one most of the work is done already. Thank you!

**Comment by gagnieray** [comment]
> Also, there is a bullet list displayed, I'm not sure the "fomantic" way to get that fixed

It's already fixed. The fomantic theme has been modified on this one.
You must have forgotten to run `npm run gulp && npm run gulp-ui build`

**Comment by gagnieray** [comment]
> not on core fields configuration. On that page, it's not possible to reorder fields, only fieldsets are.

Well, it was working at first 😅

I'll have a look at it.

**Comment by trasher** [comment]
> It's already fixed. The fomantic theme has been modified on this one. You must have forgotten to run `npm run gulp && npm run gulp-ui build`

Oh... Indeed this is OK now! I've used the `bin/install_deps` script but I suspect I had an issue locally (problems with merging your other PR). Sorry for the noise ;)

**Pull request merged**